terra_layer/utils.py

```diff
@@ -19,11 +19,10 @@ def dict_merge(dct, merge_dct, add_keys=True):
     return dct
 
 
-def get_layer_group_cache_key(pk):
+def create_layer_group_cache_key(slug):
     """
-    :param pk: The pk of the layer group to be cached
-    :type pk: int
+    :param slug: The scene slug to be cached
     :return: The cache key
     :rtype: string
     """
-    return 'terra-layer-{}'.format(pk)
+    return 'terra-layer-{}'.format(slug)
```

Comment:
Maybe here you can compose a more complex key.
Also, in terra apps we decided to do not use .format(), but fstring when it's possible.

Reply:
Tell me if it's better

Reply:
verry good :+1: